orderer/consensus/etcdraft/util.go

```diff
@@ -197,8 +200,10 @@ func ConsensusMetadataFromConfigBlock(block *common.Block) (*etcdraft.ConfigMeta
 	return MetadataFromConfigUpdate(configUpdate)
 }
 
-// CheckConfigMetadata validates Raft config metadata
-func CheckConfigMetadata(metadata *etcdraft.ConfigMetadata) error {
+// VerifyConfigMetadata validates Raft config metadata.
+// If x509.VerifyOpts is nil, it will do only sanity check of certificates.
+// If ignoreCertExpiration is true, it will verify certificate and ignore expiration errors.
+func VerifyConfigMetadata(metadata *etcdraft.ConfigMetadata, verifyOpts *x509.VerifyOptions, ignoreCertExpiration bool) error {
```

Comment:
Why would we want to ignore expiration?

Reply:
Some consenters, for which my org is not responsible for, could have expired certificates. The responsible org rotates certificates, but why should I wait for it?  It should be possible to do a config update if we have enough nodes for a quorum.

Reply:
so If I take into account expiration of certificates, this usecase fails. 
https://github.com/hyperledger/fabric/blob/master/integration/raft/cft_test.go#L519

Reply:
In that case, I think it makes sense to only look at the delta (new consenters added) and not at all certificates.

Reply:
`VerifyConfigMetadata` is used in `consenter.IsChannelMemeber`, there is no new config, it detects whether it implies that this orderer is a member of the channel or no. We shouldn't take into account expiration there. 
Delta is validated only during config update, [here](https://github.com/hyperledger/fabric/pull/1888/files#diff-9f34970b484b953e4f5ff2b4b98397e9R1344).

Reply:
maybe I can add `MembershipChanges` as optional param to `VerifyConfigMetadata`, in that way I can verify expiration only on new consenters if there are any.